recipes/configs/llama2/7B_lora_single_device.yaml

```diff
@@ -82,3 +82,9 @@ log_every_n_steps: null
 device: cuda
 dtype: bf16
 enable_activation_checkpointing: True
+
+# Show case the usage of pytorch profiler
```

Comment:
Would just comment this out and add this piece of documentation in the code itself.

Reply:
I agree with commenting it out, probably don't want it turned on by default. But won't the `config.instantiate(cfg.perf_profiler)` error out in the recipe as written then?

Reply:
Yeah, as @ebsmothers mentioned, comment this out in yaml will cause recipe error. That's the reason I created a dedicate showcase recipe for all the non-essential perf tools stuff

Reply:
ah, this makes sense. 

How about an argument `enabled` that `perf_profiler` accepts? Then we can easily include these in configs / recipes, and if enabled is False, we just don't enable the profiler. cc @ebsmothers , does this make sense?